urls.go

```diff
@@ -48,7 +48,17 @@ func (r *Repo) getMirrorlistURLs() ([]string, error) {
 		return r.URLs, nil
 	}
 
-	r.LastMirrorlistCheck = time.Now()
+	r.MirrorlistMutex.Lock()
+	defer r.MirrorlistMutex.Unlock()
+
+	// Test time again in case another routine already checked in the meantime
+	if time.Since(r.LastMirrorlistCheck) < 5*time.Second {
```

Comment:
`5*time.Second` repeats twise, let's move it to a constant with human readable name.

Reply:
Sure